src/BenchmarksDriver/Program.cs

```diff
@@ -107,6 +107,9 @@ public static int Main(string[] args)
             var kestrelThreadCountOption = app.Option("--kestrelThreadCount",
                 "Maps to KestrelServerOptions.ThreadCount.",
                 CommandOptionType.SingleValue);
+            var kestrelThreadPoolDispatchingOption = app.Option("--kestrelThreadPoolDispatching",
+                "Maps to InternalKestrelServerOptions.ThreadPoolDispatching.",
```

Comment:
I don't think it's necessary, since anyone setting this should know how the Kestrel feature works.  Also, if the Kestrel default changes someday I don't want to require changes to Benchmarks.